### PR TITLE
Adding includes issue

### LINKED ISSUE #235: Folder names vs keywords (plural and singlar)
Add to instructor notes that remembering when to use "includes" vs "include" can be a sticking point for leaners and to be indentation about pointing out  `_includes` is the folder name and `include` lets you reference something in that folder.

### PR DESCRIPTION
**Pull request by sstevens2**
Added a common issue with includes/include to instructor notes

Fixes #235

### CODE REVIEW

**Review by anenadic — COMMENTED** [review]
Did you mean 'from that folder' in the text @sstevens2? Otherwise looks good to me.

File: _extras/guide.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ title: "Instructor Notes"
   The advantage of opening a plain `.html` file in the browser is that it
   clearly demonstrates the utility of the `title` element.
 
+## Common Issues
```
Comment (tobyhodges) [inline _extras/guide.md]: I like Instructor Notes that are presented episode-by-episode. Can we change the heading to match the title of the episode this note refers to?

Comment (sstevens2) [reply _extras/guide.md]:
```suggestion
## Reusing Blocks of Content
```

File: _extras/guide.md [hunk 2]
```diff
@@ -29,6 +29,10 @@ title: "Instructor Notes"
   The advantage of opening a plain `.html` file in the browser is that it
   clearly demonstrates the utility of the `title` element.
 
+## Reusing Blocks of Content
 
+- Learners sometimes get confused about when to use "includes" vs "include".  Good to stress that the folder is `_includes`
+      and when you want to use something for that folder you `include` it.
```
Comment (sstevens2) [inline _extras/guide.md]:
```suggestion
      and when you want to use something from that folder you `include` it.
```

**Comment by sstevens2** [comment]
Good suggestions for changes @tobyhodges and @anenadic!  Think I fixed them both.

**Comment by tobyhodges** [comment]
Thank you @sstevens2!

**Pull request merged**